build/common.sh

```diff
@@ -272,7 +272,7 @@ function kube::build::docker_image_exists() {
 
   # We cannot just specify the IMAGE here as `docker images` doesn't behave as
   # expected.  See: https://github.com/docker/docker/issues/8048
-  "${DOCKER[@]}" images | grep -Eq "^${1}\s+${2}\s+"
+  "${DOCKER[@]}" images | grep -Eq "${1}\s+${2}\s+"
```

Comment:
Can you show me a line that docker emits that doesn't match the anchored regex?  I am trying to understand what changed...